### PR TITLE
Update noderesourcetopology README.md with information about scoringStrategies

### PR DESCRIPTION
**Pull request by PiotrProkop**
#### What type of PR is this?
/kind documentation

#### What this PR does / why we need it:

Updates documentation for  noderesourcetopology plugin.
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Comment by PiotrProkop** [comment]
@ffromani @swatisehgal PTAL

File: pkg/noderesourcetopology/README.md [hunk 1]
```diff
@@ -98,6 +98,24 @@ profiles:
       cacheResyncPeriodSeconds: 5
 ```
 
+#### ScoringStrategy
+
+Topology-aware scheduler supports four scoring strategies. You can set a strategy via SchedulerConfigConfiguration, by setting `scoringStrategy` option. There are four supported strategies:
```
Comment (ffromani) [inline pkg/noderesourcetopology/README.md:103]: I'd write: "The topology-aware scheduler supports four scoring strategies. You can set a strategy via SchedulerConfigConfiguration, by setting the `scoringStrategy` option."

File: pkg/noderesourcetopology/README.md [hunk 2]
```diff
@@ -98,6 +98,24 @@ profiles:
...
+
+* MostAllocated
+* BalancedAllocation
+* LeastAllocated
+* LeastNUMANodes
+
+MostAllocated, BalancedAllocation and LeastAllocated strategies work only for `single-numa-node` Topology Manager policy and indicate how score of the worker node will be calculated based
```
Comment (ffromani) [inline pkg/noderesourcetopology/README.md:110]: I'd write: "The MostAllocated, BalancedAllocation and LeastAllocated strategies only work with the `single-numa-node` Topology Manager policy"

File: pkg/noderesourcetopology/README.md [hunk 3]
```diff
@@ -98,6 +98,24 @@ profiles:
...
+on current utilization:
+
+* MostAllocated - favors node with the least amount of available resources
+* BalancedAllocation - favors node with balanced resource usage rate
+* LeastAllocated - favors node with the most amount of available resource
+
+**LeastNUMANodes** strategy works for all Topology Manager policies and favors nodes which require least amount of topology zones to satisfy resource requests for given pod.
```
Comment (ffromani) [inline pkg/noderesourcetopology/README.md:117]: "The LeastNUMANodes strategy works with all the Topology Manager policies and favors nodes which require the least amount of topology zones to satisfy the resource requests for a given pod."

**Review by ffromani — COMMENTED** [review]
Some possible wording improvements, everything else LGTM

**Comment by PiotrProkop** [comment]
> Some possible wording improvements, everything else LGTM

Thanks! I'll apply suggestions right now.

File: pkg/noderesourcetopology/README.md (hunk 1 above)
Comment (PiotrProkop) [reply pkg/noderesourcetopology/README.md]: 👍

File: pkg/noderesourcetopology/README.md (hunk 2 above)
Comment (PiotrProkop) [reply pkg/noderesourcetopology/README.md]: 👍

File: pkg/noderesourcetopology/README.md (hunk 3 above)
Comment (PiotrProkop) [reply pkg/noderesourcetopology/README.md]: 👍

**Review by ffromani — COMMENTED** [review]
/lgtm

**Review by swatisehgal — APPROVED** [review]
/lgtm

/assign @Huang-Wei 
for approval

**Pull request merged**